docker-build.sh

```diff
@@ -179,7 +178,19 @@ prepare_extra_amd64() {
     popd
     popd
     popd
+
+    # Download and install libva-utils
```

Comment:
LGTM but can you move this field to right after the `# Download and install libva`? The relationship between their version numbers seems to be very close.

Reply:
Done. I also removed the explicit version of libva-utils since we use master of libva too.